snmp.yml

```diff
@@ -824,3 +824,400 @@ cisco_wlc:
+      oid: 1.3.6.1.4.1.318.1.1.1.2.2.1
+    - name: upsAdvBatteryTemperature
+      oid: 1.3.6.1.4.1.318.1.1.1.2.2.2
+    - name: upsAdvBatteryRunTimeRemaining
+      oid: 1.3.6.1.4.1.318.1.1.1.2.2.3
+    - name: upsAdvBatteryReplaceIndicator
+      oid: 1.3.6.1.4.1.318.1.1.1.2.2.4
+    - name: upsAdvBatteryActualVoltage
+      oid: 1.3.6.1.4.1.318.1.1.1.2.2.8
+    - name: upsHighPrecBatteryCapacity
+      oid: 1.3.6.1.4.1.318.1.1.1.2.3.1
+    - name: upsHighPrecBatteryTemperature
+      oid: 1.3.6.1.4.1.318.1.1.1.2.3.2
+    - name: upsHighPrecBatteryActualVoltage
+      oid: 1.3.6.1.4.1.318.1.1.1.2.3.4
+    - name: upsBasicInputPhase
+      oid: 1.3.6.1.4.1.318.1.1.1.3.1.1
+    - name: upsAdvInputLineVoltage
+      oid: 1.3.6.1.4.1.318.1.1.1.3.2.1
+    - name: upsAdvInputMaxLineVoltage
+      oid: 1.3.6.1.4.1.318.1.1.1.3.2.2
+    - name: upsAdvInputMinLineVoltage
+      oid: 1.3.6.1.4.1.318.1.1.1.3.2.3
+    - name: upsAdvInputFrequency
+      oid: 1.3.6.1.4.1.318.1.1.1.3.2.4
+    - name: upsAdvInputLineFailCause
+      oid: 1.3.6.1.4.1.318.1.1.1.3.2.5
+    - name: upsHighPrecInputLineVoltage
+      oid: 1.3.6.1.4.1.318.1.1.1.3.3.1
+    - name: upsHighPrecInputMaxLineVoltage
+      oid: 1.3.6.1.4.1.318.1.1.1.3.3.2
+    - name: upsHighPrecInputMinLineVoltage
+      oid: 1.3.6.1.4.1.318.1.1.1.3.3.3
+    - name: upsHighPrecInputFrequency
+      oid: 1.3.6.1.4.1.318.1.1.1.3.3.4
+    - name: upsBasicOutputStatus
+      oid: 1.3.6.1.4.1.318.1.1.1.4.1.1
+    - name: upsBasicOutputPhase
+      oid: 1.3.6.1.4.1.318.1.1.1.4.1.2
+    - name: upsAdvOutputVoltage
+      oid: 1.3.6.1.4.1.318.1.1.1.4.2.1
+    - name: upsAdvOutputFrequency
+      oid: 1.3.6.1.4.1.318.1.1.1.4.2.2
+    - name: upsAdvOutputLoad
+      oid: 1.3.6.1.4.1.318.1.1.1.4.2.3
+    - name: upsAdvOutputCurrent
+      oid: 1.3.6.1.4.1.318.1.1.1.4.2.4
+    - name: upsAdvOutputKVACapacity
+      oid: 1.3.6.1.4.1.318.1.1.1.4.2.6
+    - name: upsAdvOutputActivePower
+      oid: 1.3.6.1.4.1.318.1.1.1.4.2.8
+    - name: upsAdvOutputApparentPower
+      oid: 1.3.6.1.4.1.318.1.1.1.4.2.9
+    - name: upsHighPrecOutputVoltage
+      oid: 1.3.6.1.4.1.318.1.1.1.4.3.1
+    - name: upsHighPrecOutputFrequency
+      oid: 1.3.6.1.4.1.318.1.1.1.4.3.2
+    - name: upsHighPrecOutputLoad
+      oid: 1.3.6.1.4.1.318.1.1.1.4.3.3
+    - name: upsHighPrecOutputCurrent
+      oid: 1.3.6.1.4.1.318.1.1.1.4.3.4
+    - name: upsHighPrecOutputEfficiency
+      oid: 1.3.6.1.4.1.318.1.1.1.4.3.5
+    - name: upsHighPrecOutputEnergyUsage
+      oid: 1.3.6.1.4.1.318.1.1.1.4.3.6
+    - name: upsAdvTestDiagnosticSchedule
+      oid: 1.3.6.1.4.1.318.1.1.1.7.2.1
+    - name: upsAdvTestDiagnostics
+      oid: 1.3.6.1.4.1.318.1.1.1.7.2.2
+    - name: upsAdvTestDiagnosticsResults
+      oid: 1.3.6.1.4.1.318.1.1.1.7.2.3
+    - name: upsCommStatus
+      oid: 1.3.6.1.4.1.318.1.1.1.8.1
+    - name: upsOutletGroupStatusGroupState
+      oid: 1.3.6.1.4.1.318.1.1.1.12.1.2.1.3
+      indexes:
+        - labelname: outlet_group_name
```

Comment:
upsOutletGroupStatusName would be the name of this give that's what you're looking up